pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1206,14 +1206,31 @@ func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, error) {
 					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
 					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
 				})
+			case dvmp.Status.PodPhase == corev1.PodPending:
+				t.Owner.Status.PendingPods = append(t.Owner.Status.PendingPods, &migapi.PodProgress{
+					ObjectReference:             objRef,
+					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
+					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
+				})
+				pendingMessage += dvmp.Status.LogMessage + " "
```

Comment:
So this is grabbing the logs from the status field and setting it as a warning message in the status condition? is that right?

Reply:
Yes! Since there could be multiple pods (each for one dvmp), that might be in the pending state for DVM, to me aggregating logMessage from each such dvmp and raising one warning in DVM makes more sense.

Reply:
Okay, my only concern is wouldn't this mean if I have multiple pods that are pending it will look like one long log message but will really be multiple pods logs?

Could we separate the logs with a line indicating its a new pod? Or am I missing something?

Reply:
good catch! starting new log from new line. There is only one logMessage per pod for this situation, so starting each from new line would make it less confusing.